utils/RunONNXModel.py

```diff
@@ -861,7 +867,19 @@ def main():
                         "using atol={}, rtol={} ...".format(args.atol, args.rtol),
                     )
                     verify_outs(outs[i], ref_outs[i])
-
-
+        return outs
+
+# Python function inteface for RunONNXModel
+# Arguments are passed as named parameters for the function
+# Extra functionality is to directly pass a list of arrays as inference input
+def onnxmlirrun(onnx_model=None, compiled_so=None, inputs=None):
+    if onnx_model :
+        args.model = onnx_model
+    if compiled_so :
+        args.load_so = compiled_so
```

Comment:
Should we check if both `onnx_model` and `compiled_so` are not given?

Reply:
Make sense, and maybe if none are given, an error?

Reply:
I added the check whether both onnx_mdel and compiled_so are given or not give (XOR).